ui/app/src/Utils/Match.elm

```diff
@@ -1,9 +1,32 @@
-module Utils.Match exposing (consecutiveChars, jaro, jaroWinkler)
+module Utils.Match exposing (consecutiveChars, filter, jaro, jaroWinkler)
 
 import Char
+import Regex
 import Utils.List exposing (zip)
 
 
+filter : String -> List { a | name : String } -> List { a | name : String }
+filter search list =
+    let
+        re =
+            "(.+)?"
+                ++ (String.split "" search |> String.join "(.+)?")
```

Comment:
What if search contains meaningful regex characters, e.g. `.`?

Reply:
That's a good point. If someone wants to use the receiver bar as a regex, for example `mysql-.*`, then this will create an unexpected regex.